Server/Utilities/SpringBoardAlerts.m

```diff
@@ -103,305 +105,26 @@ - (SpringBoardAlert *)alertMatchingTitle:(NSString *)alertTitle {
-    alert(@"OK", YES, @"vorrebbe inviarti delle notifiche"),
-    alert(@"OK", YES, @"vorrebbe inviarti notifiche"),
-    alert(@"OK", YES, @"desodera accedere all'opzione di riconoscimento vocale"),
-    alert(@"OK", YES, @"desidera accedere ai dati di “Movimento e fitness”"),
-    alert(@"OK", YES, @"vorrebbe accedere agli account Twitter"),
-    alert(@"OK", YES, @"desidera accedere all'opzione di riconoscimento vocale")
-  ];
-}
-
-- (NSArray<SpringBoardAlert *> *)HebrewAlerts {
-  return @[
-    alert(@"אפשר", YES, @"ת לפרטי מיקומך בעת שימושך בייש"),
-    alert(@"אפשר", YES, @"לפרטי מיקומך גם כשאינך משתמש/ת בייש"),
-    alert(@"אפשר", YES, @"ת לפרטי מיקומך אפילו כשאינך משתמש/ת בייש"),
-    alert(@"אפשר", YES, @"לגשת לפרטי מיקומך גם כשאינך משתמש/ת בייש"),
-    alert(@"אישור", YES, @"קש לגשת לאנשי הקשר ש"),
-    alert(@"אישור", YES, @"וניין לגשת ללוח השנה ש"),
-    alert(@"אישור", YES, @"וניין לגשת למשימות ש"),
-    alert(@"אישור", YES, @"וניין לגשת לתמונות ש"),
-    alert(@"אישור", YES, @"וניין לגשת אל פעילות התנועה והכושר ש"),
-    alert(@"אישור", YES, @"מבקש להשתמש במצל"),
-    alert(@"אישור", YES, @"מבקש לגשת לחשבונ"),
-    alert(@"אישור", YES, @"מעוניין לאפשר להתקני Bluetooth קרובים לקבל גישה לנתונים אפילו כשאינך משתמש/ת ביישום"),
-    alert(@"אישור", YES, @"מעוניין לשלוח לך עדכונים"),
-    alert(@"אישור", YES, @"מעוניין לגשת אל זיהוי הדיבור ש")
-  ];
-}
-
-- (NSArray<SpringBoardAlert *> *)GreekAlerts {
-  return @[
-    alert(@"Να επιτρέπεται", YES, @"η πρόσβαση στην τοποθεσία σας"),
-    alert(@"Ναι", YES, @"για να σας στέλνει γνωστοποιήσεις")
-  ];
-}
-
-- (NSArray<SpringBoardAlert *> *)HungarianAlerts {
-  return @[
-    alert(@"Engedélyezés", YES, @"hozzáférjen az Ön helyzetéhez az alkalmazás használatakor"),
-    alert(@"Engedélyezés", YES, @"értesítéseket szeretne Önnek küldeni"),
-    alert(@"Engedélyezés mindig", YES, @"számára, hogy hozzáférjen az Ön helyzetéhez")
-  ];
+    NSDataAsset *asset = [[NSDataAsset alloc] initWithName:@"alerts"];
+    NSDictionary * languages = [NSJSONSerialization JSONObjectWithData:[asset data] options:kNilOptions error:nil];
+
+    NSMutableArray<SpringBoardAlert *> * result = [NSMutableArray<SpringBoardAlert *> array];
+    NSEnumerator * languagesEnumerator = [languages keyEnumerator];
+    id language;
+    
+    while ((language = [languagesEnumerator nextObject])) {
+        NSArray * alerts = [languages objectForKey: language];
+        for (int i=0; i < alerts.count; i++) {
+            NSDictionary* alertDict = alerts[i];
+            NSString * title = [alertDict objectForKey:@"title"];
+            NSArray * buttons = [alertDict objectForKey:@"buttons"];
+            id shouldAccept = [alertDict objectForKey: @"shouldAccept"];
+            if (title != nil && shouldAccept != nil && buttons != nil && buttons.count > 0 ){
+                [result addObject: alert(buttons[0], shouldAccept, title)];
```

Comment:
I think we should produce error if JSON contains object that doesn't satisfy these checks.